Analysis/Tutorials/src/histogramRegistry.cxx

```diff
@@ -257,24 +257,22 @@ struct FTask {
   }
 };
 
-struct GTask {
+struct ConfigurablesTest {
   HistogramRegistry histos{"Histos"};
 
   // for testing: assume first element in vector is number of bins; if it is zero variable binning is assumed
   // note that the size of the default vector fixes the size of the vector that can be provided via configurable
-  Configurable<std::vector<double>> ptBinning{"pt-bin-edges", {0., 0.15, 1., 5., 10., 50.}, ""}; // variable bin edges
-  Configurable<std::vector<double>> centBinning{"cent-binning", {9., 0., 90}, ""};               // fixed size bins
+  ConfigurableAxis<double> ptBinning{"pt-bin-edges", {0., 0.15, 1., 5., 10., 50.}, ""}; // variable bin edges
+  ConfigurableAxis<double> centBinning{"cent-binning", {9, 0., 90}, ""};                // fixed size bins
```

Comment:
@aalkin @saganatt Shall we make a constant / enum value VARIABLE_WIDTH to make it clear (instead of the 0. )?

Reply:
Sure, done